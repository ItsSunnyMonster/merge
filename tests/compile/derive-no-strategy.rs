use merge::Merge;

#[derive(Merge)]
struct S {
    #[merge(strategy)]
    field1: u8,
}

fn main() {}
