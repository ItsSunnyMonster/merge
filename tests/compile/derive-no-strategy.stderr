error: expected `=`
 --> tests/compile/derive-no-strategy.rs:5:21
  |
5 |     #[merge(strategy)]
  |                     ^
